app/api/core/service/content_filtering.rb

```diff
@@ -31,11 +31,7 @@ def process_request_body
     end
 
     def process_response_body
-      headers('Content-Type' => request_accepted)
-    end
-
-    def process_json_response_body
```

Comment:
Was this part of the upgrade?

Reply:
Yes. Rack changed its handling of arrays in Content-Type headers. The removal of process_json_response_body was just because it was unused code.